### PR TITLE
fix(perf-nemesis-latency): Add operation duration to report

### PR DESCRIPTION
**Pull request by aleksbykov**
Add new column, which contain operation duration in seconds

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by aleksbykov** [comment]
Screenshot from 2023-02-22 20-47-21

File: sdcm/report_templates/results_latency_during_ops_short.html [hunk 1]
```diff
@@ -78,21 +78,24 @@ <h2>{{ operation }}</h2>
...
-                                            <td rowspan="{{ row_span }}"> Cycle #{{ loop.index }} </td>
+                                            <td rowspan="{{ row_span }}"> Cycle #{{ cycle_index }} </td>
                                             <td rowspan="{{ row_span }}"> current build </td>
                                             <td> {{ workload }}</td>
                                             {% for perc in lat_type_list %}
-                                                <td style="background-color: {{ cycle['hdr_summary'][workload]['color'][perc] }}"" }};"> {{ cycle['hdr_summary'][workload][perc]}} </td>
+                                                <td style="background-color: {{ cycle['hdr_summary'][workload]['color'][perc] }};"> {{ cycle['hdr_summary'][workload][perc]}} </td>
                                                 <td> {{ stats["Steady State"]["hdr_summary"][workload][perc] }} </td>
                                             {% endfor %}
+                                            <td rowspan="{{ row_span }}"> {{ (cycle['hdr_summary'][workload]['end_time'] - cycle['hdr_summary'][workload]['start_time']) // 1000 }} </td>
```
Comment (fgelcer) [inline sdcm/report_templates/results_latency_during_ops_short.html:98]: it looks fine, but isn't it better to calculate in the Python code? what will happen if, for some reason, these will not be found?

File: sdcm/report_templates/results_latency_during_ops_short.html [hunk 2]
```diff
@@ -78,21 +78,24 @@ <h2>{{ operation }}</h2>
...
                             <th>Commit id, date</th>
                         </tr>
                         {% for cycle in results["cycles"] %}
                             {% set row_span = cycle["hdr_summary"]|length %}
+                            {% set cycle_index = loop.index %}
                             {% for workload in cycle["hdr_summary"] %}
                                 <tr>
                                     {% if loop.first %}
-                                            <td rowspan="{{ row_span }}"> Cycle #{{ loop.index }} </td>
+                                            <td rowspan="{{ row_span }}"> Cycle #{{ cycle_index }} </td>
```
Comment (fgelcer) [inline sdcm/report_templates/results_latency_during_ops_short.html:91]: this is the fix for the index showing always number 1 cycle, right?

File: sdcm/report_templates/results_latency_during_ops_short.html [hunk 3]
```diff
@@ -78,21 +78,24 @@ <h2>{{ operation }}</h2>
...
                                             <td rowspan="{{ row_span }}"> current build </td>
                                             <td> {{ workload }}</td>
                                             {% for perc in lat_type_list %}
-                                                <td style="background-color: {{ cycle['hdr_summary'][workload]['color'][perc] }}"" }};"> {{ cycle['hdr_summary'][workload][perc]}} </td>
+                                                <td style="background-color: {{ cycle['hdr_summary'][workload]['color'][perc] }};"> {{ cycle['hdr_summary'][workload][perc]}} </td>
```
Comment (fgelcer) [inline sdcm/report_templates/results_latency_during_ops_short.html:95]: what is this change about? the table's header?

**Review by roydahan — APPROVED** [review]

File: sdcm/report_templates/results_latency_during_ops_short.html (hunk 1 above)
Comment (roydahan) [reply sdcm/report_templates/results_latency_during_ops_short.html:98]: Then we won't have many other things.

Comment (fgelcer) [reply sdcm/report_templates/results_latency_during_ops_short.html:98]: true... @aleksbykov , are you doing the calculation in Python at the end? or we leave it as is?

File: sdcm/report_templates/results_latency_during_ops_short.html (hunk 2 above)
Comment (aleksbykov) [reply sdcm/report_templates/results_latency_during_ops_short.html:91]: yes

File: sdcm/report_templates/results_latency_during_ops_short.html (hunk 3 above)
Comment (aleksbykov) [reply sdcm/report_templates/results_latency_during_ops_short.html]: this is fix of invalid syntax

**Comment by aleksbykov** [comment]
@fgelcer @roydahan , Could review one more time? I moved calculation to python code

**Review by fgelcer — APPROVED** [review]
LGTM

**Pull request merged**